### PR TITLE
php-cs-fixer config

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: tools/cs-fixer/.php_cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ->exclude(array('Tests/Fixtures'))
+;
+
+return Symfony\CS\Config\Config::create()
+    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
+    ->fixers(array(
+        'header_comment',
+        '-unalign_double_arrow',
+        '-unalign_equals',
+        'align_double_arrow',
```
Comment (greg0ire) [inline tools/cs-fixer/.php_cs]: 23 and 25 conflict.

File: tools/cs-fixer/.php_cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ->in(array(__DIR__))
+    ->exclude(array('Tests/Fixtures'))
+;
+
+return Symfony\CS\Config\Config::create()
+    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
+    ->fixers(array(
+        'header_comment',
+        '-unalign_double_arrow',
+        '-unalign_equals',
```
Comment (greg0ire) [inline tools/cs-fixer/.php_cs]: I think we should drop alignement, it makes diff harder to read, makes conflicts easier to get (more code changed for nothing).

Comment (greg0ire) [inline tools/cs-fixer/.php_cs]: See https://github.com/sonata-project/SonataAdminBundle/pull/3647#discussion_r60448093

**Comment by Soullivaneuh** [comment]
@core23 I'm closing this one because we will integrate StyleCI tool soon.

**Comment by core23** [comment]
@Soullivaneuh I'm not firm with the StyleCI tool, but shouldn't we leave this config in the repository for the developers that exectutes the fixer locally?

**Comment by Soullivaneuh** [comment]
@core23 https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge :-)

**Comment by Soullivaneuh** [comment]
Please see #9.

**Comment by core23** [comment]
So we add a ``.php_cs`` file that will use the config from ``.styleci.yml``? Who executes the cs fixer? The user or the StyleCI service?

**Comment by Soullivaneuh** [comment]
> So we add a .php_cs file that will use the config from .styleci.yml?

Exactly.

>  Who executes the cs fixer? The user or the StyleCI service?

Both.

The StyleCI service only care of `.style-ci.yml` file.

The user have nothing to change of his habits: `php-cs-fixer run --verbose`.

The bridge is just here to permit contributors to maintain only one file: `.styleci.yml`.

This avoid duplicate and errors.

**Pull request closed without merging**